### PR TITLE
Tai 2099 Play 2.5.12 upgrade

### PR DESCRIPTION
**Pull request by shane-wilson**

### CODE REVIEW

File: conf/application-json-logger.xml [hunk 1]
```diff
@@ -5,9 +5,9 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
-    <logger name="uk.gov" level="DEBUG"/>
+    <logger name="uk.gov" level="WARN"/>
```
Comment (ShaolinSarg) [inline conf/application-json-logger.xml]: Can we make the config value the same as the way its defined in the bbsi project?
```
<logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
<root level="${logger.application:-WARN}">
```

File: project/FrontendBuild.scala [hunk 2]
```diff
@@ -49,16 +49,18 @@ private object AppDependencies {
...
-        "org.pegdown" % "pegdown" % pegdownVersion % scope,
-        "org.jsoup" % "jsoup" % "1.8.1" % scope,
-        "com.typesafe.play" %% "play-test" % PlayVersion.current % scope
+        "org.scalatestplus.play" %% "scalatestplus-play" % "1.5.1" % scope,
+        "org.pegdown" % "pegdown" % pegdownVersion % scope,//
+        "org.jsoup" % "jsoup" % "1.8.3" % scope,//
+        "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,//
+        "org.mockito" % "mockito-core" % "1.9.0" % scope,
+        "org.mockito" % "mockito-all" % "1.9.5" % scope//
       )
```
Comment (ShaolinSarg) [inline project/FrontendBuild.scala]: can you remove the comments from the end of the line please

File: project/FrontendBuild.scala [hunk 3]
```diff
@@ -8,35 +8,35 @@ object FrontendBuild extends Build with MicroService {
...
-  private val playUiVersion = "4.17.2"
-  private val playPartialsVersion = "4.6.0"
-  private val playAuthorisedFrontendVersion = "5.8.0"
-  private val playConfigVersion = "2.1.0"
-  private val hmrcTestVersion = "1.8.0"
+  private val playHealthVersion = "2.1.0"
+  private val logbackJsonLoggerVersion = "3.1.0"
+  private val frontendBootstrapVersion = "7.14.0"
+  private val govukTemplateVersion = "5.1.0"
+  private val playUiVersion = "7.0.0"
```
Comment (timsb) [inline project/FrontendBuild.scala]: @shane-wilson I've read that there were some errors with the version of Play-ui. Could you upgrade to the latest version which at the moment is 7.2.0

File: project/FrontendBuild.scala [hunk 4]
```diff
@@ -8,35 +8,35 @@ object FrontendBuild extends Build with MicroService {
...
     "uk.gov.hmrc" %% "play-partials" % playPartialsVersion,
     "uk.gov.hmrc" %% "play-authorised-frontend" % playAuthorisedFrontendVersion,
     "uk.gov.hmrc" %% "play-config" % playConfigVersion,
-    "uk.gov.hmrc" %% "play-json-logger" % playJsonLoggerVersion,
+    "uk.gov.hmrc" %% "logback-json-logger" % logbackJsonLoggerVersion,
     "uk.gov.hmrc" %% "govuk-template" % govukTemplateVersion,
     "uk.gov.hmrc" %% "play-health" % playHealthVersion,
     "uk.gov.hmrc" %% "play-ui" % playUiVersion,
-    "uk.gov.hmrc" %% "url-builder" % urlBuilderVersion,
+    "uk.gov.hmrc" %% "url-builder" % urlBuilderVersion, //
```
Comment (timsb) [inline project/FrontendBuild.scala]: @shane-wilson Could you remove this comment please?

**Pull request merged**